astropy/cosmology/__init__.py

```diff
@@ -8,9 +8,10 @@
 
 """
 
-from . import core, funcs, realizations
+from . import core, funcs, realizations, utils
 from .core import *
 from .funcs import *
 from .realizations import *
+from .utils import *
```

Comment:
I'd not be this ready for future expansion, but now I'm in full-on nitpick territory!

Reply:
I'm not sure what, if anything, will be in utils and publicly scoped. But I like having this here because I've purposefully put a blank ``__all__`` in ``utils`` so that if anything is ever publicly scoped, it is automatically moved up to ``cosmology``.